.github/workflows/lint.yml

```diff
@@ -78,22 +78,27 @@ jobs:
         if: env.GIT_DIFF
       - uses: golangci/golangci-lint-action@v3
         with:
+          version: v1.52
```

Comment:
Reported issues using latest version of golanci (v1.53) so using previous for now.